requests_toolbelt/_compat.py

```diff
@@ -18,11 +18,13 @@
     from requests.packages.urllib3 import fields
     from requests.packages.urllib3 import filepost
     from requests.packages.urllib3 import poolmanager
+    from requests.packages.urllib3.util import timeout
```

Comment:
This fallback needs to be its own try/except block because it's causing the normal tests to fail.